cmd/pg/backup_fetch.go

```diff
@@ -12,25 +13,36 @@ const (
 matches given shell file pattern.
 For information about pattern syntax view: https://golang.org/pkg/path/filepath/#Match`
 	RestoreSpecDescription = "Path to file containing tablespace restore specification"
+	ReverseDeltaUnpackDescription = "Unpack delta backups in reverse order (testing)"
 )
 
 var fileMask string
 var restoreSpec string
+var reverseDeltaUnpack bool
 
-// backupFetchCmd represents the backupFetch command
 var backupFetchCmd = &cobra.Command{
 	Use:   "backup-fetch destination_directory backup_name",
 	Short: BackupFetchShortDescription, // TODO : improve description
 	Args:  cobra.ExactArgs(2),
 	Run: func(cmd *cobra.Command, args []string) {
 		folder, err := internal.ConfigureFolder()
 		tracelog.ErrorLogger.FatalOnError(err)
-		internal.HandleBackupFetch(folder, args[1], internal.GetPgFetcher(args[0], fileMask, restoreSpec))
+
+		var pgFetcher func(folder storage.Folder, backup internal.Backup)
+		if reverseDeltaUnpack {
+			pgFetcher = internal.GetPgFetcherNew(args[0], fileMask, restoreSpec)
+		} else {
+			pgFetcher = internal.GetPgFetcherOld(args[0], fileMask, restoreSpec)
+		}
+
+		internal.HandleBackupFetch(folder, args[1], pgFetcher)
 	},
 }
 
 func init() {
 	backupFetchCmd.Flags().StringVar(&fileMask, "mask", "", MaskFlagDescription)
 	backupFetchCmd.Flags().StringVar(&restoreSpec, "restore-spec", "", RestoreSpecDescription)
+	backupFetchCmd.Flags().BoolVar(&reverseDeltaUnpack, "reverse-unpack",
```

Comment:
Lots of users use config or env variables for wal-g. 
In the next PR's add the opportunity to set using new logic by config and by env variables.